Winwheel.js

```diff
@@ -696,8 +696,13 @@ Winwheel.prototype.drawSegments = function()
                     }
                     else
                     {
-                       //++ do need to draw the starting line in the correct x + y based on the start angle
-                       //++ otherwise as seen when the wheel does not use up 360 the starting segment is missing the stroked side,
+                       // Work out the x and y values for the starting point of the segment which is at its starting angle
+                       // but out from the center point of the wheel by the value of the innerRadius. Some correction for line width is needed.
+                       var iX = Math.cos(this.degToRad(seg.startAngle + this.rotationAngle - 90)) * (this.innerRadius - lineWidth / 2);
```

Comment:
TODO: check what happens if lineWidth is 1, or even if linewidth is an odd value as canvas does not like 0.5 values at times. Just to be sure.